SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -30,6 +30,14 @@ FLocalDeploymentManager::FLocalDeploymentManager()
 	// Don't kick off background processes when running commandlets
 	if (IsRunningCommandlet() == false)
 	{
+		// Check for the existence of Spatial and Spot. If they don't exist then don't start any background processes. Disable spatial networking if either is true.
+		if (!PreRunChecks())
+		{
+			UE_LOG(LogSpatialDeploymentManager, Warning, TEXT("Pre run checks for LocalDeploymentManager failed. Local deployments cannot be started. Spatial networking will be disabled."));
+			GetMutableDefault<UGeneralProjectSettings>()->bSpatialNetworking = false;
```

Comment:
Is that what we think customers want? If LocalDeployment fails - we encourage them to not use UnrealGDK by disabling bSpatialNetworking? :)

Reply:
I believe so, if the spatial and spot binaries can't be started then attempting to run any spatial or spot command (starting local deployments) will just produce cryptic errors and users will never be able to connect as the clients/servers net drivers will be waiting for spatial to launch.

Reply:
Oh right, I thought that PreRunChecks were in TryStartSpatialService, so that when customers click Start button - it prints warning and early outs. But since it's in constructor - that make sense to switch off spatial networking to avoid confusing errors.